zipkin-storage/cassandra-v1/src/main/java/zipkin2/storage/cassandra/v1/CassandraSpanConsumer.java

```diff
@@ -91,22 +90,22 @@
       spansToIndex.add(span);
     }
 
-    List<Call<ResultSet>> calls = new ArrayList<>();
+    List<Call<Void>> calls = new ArrayList<>();
     for (InsertTrace.Input insert : insertTraces) {
       calls.add(insertTrace.create(insert));
     }
     for (String insert : insertServiceNames) {
-      calls.add(insertServiceName.create(insert));
+      insertServiceName.maybeAdd(insert, calls);
```

Comment:
before, this would fill up with calls that constantly return. now if a call is obviated a dummy call is not added